src/Utils/Encoder.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace MultiversX\Utils;
+
+use Brick\Math\BigInteger;
+use MultiversX\Address;
+
+class Encoder
+{
+    public static function toHex(string|int|BigInteger|Address $value, bool $skipHexBytePadding = false): string
```

Comment:
Perhaps users should never set `skipHexBytePadding = true`?

Reply:
Removed; stemmed from old times